### ISSUE TITLE
Codemoji Comms for World Emoji Day

### ISSUE THREAD

**Issue by HPaulJohnson** [MozillaFoundation/Advocacy#73]
Wave 2 of promotion around Codemoji is being planned around World Emoji Day on 7/17. Plan includes email, social, snippet and comms. 

This ticket is to track PR portion of plan and any other supporting Comms. 

Deliverables (tbd)

cc: @KevZawacki @damiankay

**Comment by KevZawacki** [comment]
Noting that I'm on PTO next week (July 11-15). Anticipating that, I've updated our comms plan accordingly, [here](https://docs.google.com/document/d/1d6SFYpC0XLg_ldyb_hdhIi76KBwmWjBBZJbDJvET8QM/edit#), complete with our pitch.

I've handed the plan over to MoCo Comms via Vidyo meetings / email threads this week. MoCo Comms will execute the pitching alongside their agency. That work is being tracked in Trello [here](https://trello.com/c/fr59sBi5/38-world-emoji-day-pitching) (@hpauljohnson has access).

@hpauljohnson and I will discuss hand-over plans further during today's weekly meeting.

cc @damiankay
